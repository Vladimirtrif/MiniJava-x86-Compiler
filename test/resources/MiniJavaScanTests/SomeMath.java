int a = 5;
int b = a + 5;
System.out.println(b);