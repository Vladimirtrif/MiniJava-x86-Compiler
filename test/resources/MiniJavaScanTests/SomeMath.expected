INT ID(a) BECOMES INTEGER(5) SEMICOLON INT ID(b) BECOMES ID(a) PLUS INTEGER(5) SEMICOLON PRINT LPAREN ID(b) RPAREN SEMICOLON